bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/thing/ThingTypeResource.java

```diff
@@ -101,32 +96,29 @@ public Response getByUID(@PathParam("thingTypeUID") String thingTypeUID,
-                    ConfigDescriptionParameterBean configDescriptionParameterBean = new ConfigDescriptionParameterBean(
-                            configDescriptionParameter.getName(), configDescriptionParameter.getType(),
-                            configDescriptionParameter.getMinimum(), configDescriptionParameter.getMaximum(),
-                            configDescriptionParameter.getStepSize(), configDescriptionParameter.getPattern(),
-                            configDescriptionParameter.isRequired(), configDescriptionParameter.isReadOnly(),
-                            configDescriptionParameter.isMultiple(), configDescriptionParameter.getContext(),
-                            String.valueOf(configDescriptionParameter.getDefault()),
-                            configDescriptionParameter.getLabel(), configDescriptionParameter.getDescription(),
-                            createBeansForOptions(configDescriptionParameter.getOptions()),
-                            createBeansForCriteria(configDescriptionParameter.getFilterCriteria()));
-                    configDescriptionParameterBeans.add(configDescriptionParameterBean);
-                }
-                return configDescriptionParameterBeans;
+
+        ConfigDescription configDescription = configDescriptionRegistry.getConfigDescription(configDescriptionURI, locale);
+        if (configDescription != null) {
+            List<ConfigDescriptionParameterBean> configDescriptionParameterBeans = new ArrayList<>(
+                    configDescription.getParameters().size());
+            for (ConfigDescriptionParameter configDescriptionParameter : configDescription.getParameters()) {
+                ConfigDescriptionParameterBean configDescriptionParameterBean = new ConfigDescriptionParameterBean(
+                        configDescriptionParameter.getName(), configDescriptionParameter.getType(),
+                        configDescriptionParameter.getMinimum(), configDescriptionParameter.getMaximum(),
+                        configDescriptionParameter.getStepSize(), configDescriptionParameter.getPattern(),
+                        configDescriptionParameter.isRequired(), configDescriptionParameter.isReadOnly(),
+                        configDescriptionParameter.isMultiple(), configDescriptionParameter.getContext(),
+                        String.valueOf(configDescriptionParameter.getDefault()),
+                        configDescriptionParameter.getLabel(), configDescriptionParameter.getDescription(),
+                        createBeansForOptions(configDescriptionParameter.getOptions()),
+                        createBeansForCriteria(configDescriptionParameter.getFilterCriteria()));
+                configDescriptionParameterBeans.add(configDescriptionParameterBean);
             }
-        } catch (URISyntaxException ex) {
-            logger.error(ex.getMessage(), ex);
+            return configDescriptionParameterBeans;
         }
+       
```

Comment:
OK, I will add it manually to the save actions for the moment. Thanks for the hint!